borsh/Cargo.toml

```diff
@@ -28,7 +28,7 @@ cfg_aliases = "0.1.0"
 
 [dependencies]
 borsh-derive = { path = "../borsh-derive", version = "1.0.0-alpha.3", optional = true }
-hashbrown = { version = ">=0.11,<0.14", optional = true }
+hashbrown = { version = "0.14.0", optional = true }
```

Comment:
I would just expand the list of supported versions as it seems that there is no reason to restrict use of older versions:

```suggestion
# hashbrown can be used in no-std context.
# NOTE: There is no reason to restrict use of older versions, but we don't want to get sudden breaking changes with an open range of versions, so we limit the range by not yet released 0.15.0 version:
hashbrown = { version = ">=0.11,<0.15.0", optional = true }
```